Edited content for figure 3 (#22)
greenelab/iscb-diversity-manuscript: content/20.results.md

### Predicting Nationality with LSTM Neural Networks and Wikipedia

> 

Reviewer: ```suggestion

We next aimed to predict the nationality of honorees and authors.
We constructed a training dataset with more than 700,000 name-nationality pairs by parsing English-language Wikipedia.
We trained a LSTM neural network on n-grams to predict nationality.
We found similar performance across 1, 2, and 3-grams; however, the classifier required fewer epochs to train with 3-grams so we used this length in the model that we term Wiki2019-LSTM.
```

Our Wiki2019-LSTM returns, for each given name, a probability of that name originating from each of our 8 regions.

> The classifier returned a multiclass ROC/AUC score of 0.954, indicating we were able to recapitulate name origin with high sensitivity and specificity.

Reviewer: ```suggestion
We observed a multiclass ROC/AUC score of 0.954 for the classifier, indicating that the classifier can recapitulate name origin with high sensitivity and specificity.
```

> AUC calculated for each region individually was also high, meaning that our classifier was able to perform well on all regions (Fig. {@fig:wiki2019_lstm}).

Reviewer: ```suggestion
AUC calculated for each region individually was also high (Fig. {@fig:wiki2019_lstm}), suggesting that our classifier was sufficient for use in a broad-scale examination of disparities.
```
